src/app.js

```diff
@@ -92,8 +100,8 @@ var accountTradeHandler = function (username, password, sharedSecret) {
         var message = offer.id + " ▶ " + account + " received an offer. ";
         if (offer.itemsToGive.length == 0) {
             offer.accept(function (err) {
-                if (err) {
-                    storage.push(offer);
+                if (err) { 
+                    /*storage.push(offer);*/ // disabled until fixed
```

Comment:
Don't just comment it, remove it completely. You should also remove the handler which checks the stored trades.